doc/sphinx/source/inspection/index.rst

```diff
@@ -0,0 +1,37 @@
+.. inspection/index: 
+
+Visualization Tools
+###################
+
+One option for visualizing the graph or node structure of a graph is to compile 
```

Comment:
NGRAPH_JSON_ENABLE will be enabled by default and available in our default wheels. So lets not mention it here. Instead, we could talk about how ngraph provides serialization and deserialization facilities along with the ability to create image formats.